prow/plugins/lgtm/lgtm.go

```diff
@@ -253,45 +258,73 @@ func handle(wantLGTM bool, config *plugins.Configuration, ownersClient repoowner
 	hasLGTM := false
 	labels, err := gc.GetIssueLabels(org, repoName, number)
 	if err != nil {
-		log.WithError(err).Errorf("Failed to get the labels on %s/%s#%d.", org, repoName, number)
+		log.WithError(err).Errorf("Failed to get issue labels.")
 	}
 
 	hasLGTM = github.HasLabel(LgtmLabel, labels)
 
 	if hasLGTM && !wantLGTM {
 		log.Info("Removing LGTM label.")
-		return gc.RemoveLabel(org, repoName, number, LgtmLabel)
+		if err := gc.RemoveLabel(org, repoName, number, LgtmLabel); err != nil {
+			return err
+		}
+		log.Info("Removing tree-hash comment.")
```

Comment:
Move this to where the comment is deleted. This is logged even if we do not remove a tree-hash comment.